app/components/app/app.js

```diff
@@ -127,6 +127,15 @@ var AppComponent = React.createClass({
     };
   },
 
+  initAppUtils: function(injectedUtils){
```

Comment:
yeah tests fail every-time otherwise b/c of the chrome check. Will have a look at rewire